platform-ajax/src/test/java/com/softicar/platform/ajax/dom/image/viewer/DomImageViewerTest.java

```diff
@@ -198,12 +217,47 @@ private void clickRotateButton() {
 	private void assertRotated(boolean rotated) {
 
 		if (rotated) {
-			output.assertCssTransform("matrix(-1, 0, 0, -1, 0, 0)", findImageDiv());
+			output.assertCssTransform("matrix(-1, 0, 0, -1, 0, 0)", findImageHolder());
 		} else {
-			output.assertCssTransform("none", findImageDiv());
+			output.assertCssTransform("matrix(1, 0, 0, 1, 0, 0)", findImageHolder());
```

Comment:
We're implicitly comparing double values without epsilon here. That might backfire soon.

Reply:
ok, implemented small class AjaxCssTransformAsserter to parse matrix and compare with delta